src/components/dashboard/Confirmation.js

```diff
@@ -73,7 +74,14 @@ const ReceiveConfirmation = ({ screenProps, styles, ...props }: ReceiveProps) =>
           <Section.Text style={styles.textRow}>{reason}</Section.Text>
         </Section.Stack>
         <Section.Stack>
-          <ShareButton share={share} onPressDone={screenProps.goToRoot} actionText="Share as link" />
+          <ShareButton
+            share={share}
+            onPressDone={() => {
+              fireEvent('RECEIVE', { type: 'link' })
```

Comment:
this screen is both for send and receive

Reply:
no. we have SendConfirmation and ReceiveConfirmation.
as u see on sendflow gif event is not fired
![send](https://user-images.githubusercontent.com/36693645/70451717-88cd2980-1aae-11ea-89da-96e803dd2ed4.gif)

Reply:
ok so lets refactor the name + remove unused send code